src/components/ebay-menu-button/test/__snapshots__/renders-with-overflow-variant.expected.html

```diff
@@ -12,21 +12,21 @@
     [36m>[39m
       [36m<svg[39m
         [33maria-hidden[39m=[32m"true"[39m
-        [33mclass[39m=[32m"icon icon--overflow"[39m
+        [33mclass[39m=[32m"icon icon--overflow-small"[39m
         [33mfocusable[39m=[32m"false"[39m
       [36m>[39m
         [36m<defs>[39m
           [36m<symbol[39m
-            [33mid[39m=[32m"icon-overflow"[39m
-            [33mviewBox[39m=[32m"0 0 4 18"[39m
+            [33mid[39m=[32m"icon-overflow-small"[39m
+            [33mviewBox[39m=[32m"0 0 3 13"[39m
           [36m>[39m
             [36m<path[39m
-              [33md[39m=[32m"M2 4A2 2 0 102.001.001 2 2 0 002 4zm0 7a2 2 0 10.001-3.999A2 2 0 002 11zm2 5a2 2 0 11-3.999.001A2 2 0 014 16z"[39m
+              [33md[39m=[32m"M1.5 10a1.5 1.5 0 110 3 1.5 1.5 0 010-3zm0-5a1.5 1.5 0 110 3 1.5 1.5 0 010-3zm0-5a1.5 1.5 0 110 3 1.5 1.5 0 010-3z"[39m
```

Comment:
Might be worth documenting how these snapshots are generated, and the general process around it, in our contributing guide if we haven't done so already.

Reply:
I will add a task to do for post release. Or i can add a new command to also update snapshots automatically